contributing_to_docs/doc_guidelines.adoc

```diff
@@ -122,6 +122,31 @@ To use a discrete heading, just add `[discrete]` to the line before your unique
 == Managing authorization policies
 ----
 
+=== Creating unique IDs for 'Additional resources' and 'Next steps' titles
+
+Use unique IDs for *Additional resources* and *Next steps* titles. You can append `-additional-resources` or `-next-steps` to a unique string that describes the assembly topic.
+
+The following examples include IDs that are unique to the *Understanding the monitoring stack* assembly.
+
+*Example unique ID for an 'Additional resources' title*
+
+----
+[id="understanding-the-monitoring-stack-additional-resources"]
+== Additional resources
+----
+
+*Example unique ID for a 'Next steps' title*
+
+----
+[id="understanding-the-monitoring-stack-next-steps"]
+== Next steps
+----
+
+[NOTE]
```

Comment:
I am now wondering if this note should be removed from the PR, because it contradicts the content added in [the PR that @bmcelvee mentioned](https://github.com/openshift/openshift-docs/pull/30125). This comment states not to add IDs to 'Additional resources' and 'Next steps' block titles. Then in the **Writing assemblies** section that follows there is a note that says "Do not use `.Prerequisites`, `.Next steps`, or `.Additional information`" at all.

I have been using block titles for 'Additional resources' sections that are specific to one module but not an entire assembly. There are a number of examples in [Monitoring -> Configuring the monitoring stack](https://docs.openshift.com/container-platform/4.7/monitoring/configuring-the-monitoring-stack.html).

@openshift/team-documentation should we remove this note or update the guidance in **Writing assemblies** to accommodate those types of 'Additional resources' subsection entries?

Reply:
Those module-specific **Additional resources** subsections provide readers with easily findable additional material  specific to a module. For assemblies with many modules, I think this makes sense rather than having a long list of additional resources in the main `== Additional resources` section at the end of the assembly. The main `== Additional resources` section can be used to focus on high level resources relating to the assembly overall.

Reply:
@pneedle-rh I agree with what you're saying here, but I'm a little confused because I thought https://github.com/openshift/openshift-docs/pull/30125 from @bobfuru was meant to address the confusion you're referring to (that is, that it's OK to use block titles at the assembly level when the list is relevant specifically to the module it immediately follows, and not the entire assembly). Do you think things are still unclear about what types of additional resources headings are and aren't allowed now that that PR has merged?

As for the "NOTE" below, I guess I could go either way on its inclusion. We don't necessarily _need_ there to be a unique ID added for the block titles (like we do when it's a `==` heading because of the auto-generation that happens when it's missing), but I suppose someone might want add a unique ID in some scenarios to gain that direct anchor... Though in that case it's probably a more advanced situation, so it's likely better to make it clear we don't expect block titles to have IDs.

Reply:
Could just update the note to say something like:

> Unique IDs are not required for *Additional resources* or *Next steps* subsections that use block titles, such as `.Additional resources`.